gammapy/cube/make.py

```diff
@@ -257,3 +297,359 @@ def _check_selection(selection):
+
+        """
+        return self._run(observations, sum_over_axis=False)
+
+    def significance_map(self, maplist, convolution_radius=None):
+        """
+        Parameters
+        ---------
+        maplist: the list of dictionaries returned by MapMakerRing.run()
+                or MapMakerRing.run_images()
+        convolution_radius: `~astropy.units.Quantity`
+            The convolution radius for Tophat2DKernel
+
+            If not specified, returns the uncorrelated significance map
+
+        Returns
+        -------
+        significance_map: `~gammapy.maps.Map`
+        Stacked significance map for the entire region.
+
+        """
+        if convolution_radius:
+            scale = self.geom.pixel_scales[0].to("deg")
+            theta = (convolution_radius * scale).value
+        else:
+            theta = 1
+        tophat = Tophat2DKernel(theta)
+        tophat.normalize("peak")
+        selection = ["on", "off", "exposure_on", "exposure_off"]
+
+        stacked_map = {}
+        for name in selection:
+            stacked_map[name] = Map.from_geom(maplist[0]["counts"].geom)
+
+        for amap in maplist:
+            stacked_map["on"] += amap["counts"]
+            stacked_map["off"] += amap["off"]
+            stacked_map["exposure_on"] += amap["background"]
+            stacked_map["exposure_off"] += amap["exposure_off"]
+        stacked_map["exposure_on"] = stacked_map["exposure_on"].convolve(tophat.array)
+        stacked_map["exposure_off"] = stacked_map["exposure_off"].convolve(tophat.array)
+        stacked_map["on"] = stacked_map["on"].convolve(tophat.array)
+        stacked_map["off"] = stacked_map["off"].convolve(tophat.array)
+        alpha_map = stacked_map["exposure_on"] / stacked_map["exposure_off"]
+
+        data = significance_on_off(
+            n_on=stacked_map["on"].data,
+            n_off=stacked_map["off"].data,
+            alpha=alpha_map.data,
+            method="lima",
+        )
+        return stacked_map["on"].copy(data=data)
+
+    def significance_map_off(self, maplist, convolution_radius=None):
+        """returns the significance map with exclusion region applied
+
+        Parameters
+        ---------
+        maplist: the list of dictionaries returned by MapMakerRing.run()
+                or MapMakerRing.run_images()
+        convolution_radius: `~astropy.units.Quantity`
+            The convolution radius for Tophat2DKernel
+
+            If not specified, returns the uncorrelated significance map
+
+        Returns
+        -------
+        significance_map_off: `~gammapy.maps.Map`
+        Stacked significance map for the off regions region.
+
+        """
+
+        significance_map = self.significance_map(maplist, convolution_radius)
+        if significance_map.geom.is_image:
+            return significance_map * self.exclusion_mask.get_image_by_idx([0])
+        else:
+            return significance_map * self.exclusion_mask
+
+    def excess_map(self, maplist):
```

Comment:
This method does not use `self`, in this case it should be declared as as `@staticmethod`.